make/config.mk

```diff
@@ -59,6 +59,9 @@ USE_OPENCV = 1
 # use openmp for parallelization
```

Comment:
Sorry for not being clear. I didn't mean to move MKL_README to make/. I meant picking out the relevant sentences and put them here. 
The MKLML_ROOT, USE_MKL2016 and UES_MKL_EXPERIMENTAL options should have detailed explanation of what they do. Or at least point the user to MKL_README for more details